PWGEM/PhotonMeson/Core/EMCPhotonCut.h

```diff
@@ -73,10 +75,8 @@ class EMCPhotonCut : public TNamed
   // Temporary function to check if cluster passes a given selection criteria. To be replaced by framework filters.
   // Returns true if a cluster survives the cuts!
   template <typename Cluster, typename Track>
-  bool IsSelectedEMCal(const EMCPhotonCuts& cut, Cluster const& cluster, std::optional<Track> const& track = std::nullopt) const
+  bool IsSelectedEMCal(const EMCPhotonCuts& cut, Cluster const& cluster, Track const& track) const
```

Comment:
Why change the optional part? I mean we don't always have tracks so I thought having the tracks as optional argument would be good?

Reply:
nah it should be fine, thanks